dashboard/src/app/projects/create-project/create-project.controller.js

```diff
@@ -905,8 +905,9 @@ export class CreateProjectCtrl {
   createWorkspace() {
     this.createProjectSvc.setWorkspaceOfProject(this.workspaceName);
     let attributes = this.stack ? {stackId: this.stack.id} : {};
+    let commands = this.stack && this.stack.workspaceConfig.commands ? this.stack.workspaceConfig.commands : [];
     //TODO: no account in che ? it's null when testing on localhost
-    let creationPromise = this.cheAPI.getWorkspace().createWorkspace(null, this.workspaceName, this.recipeUrl, this.workspaceRam, attributes);
+    let creationPromise = this.cheAPI.getWorkspace().createWorkspace(null, this.workspaceName, this.recipeUrl, this.workspaceRam, attributes, commands);
```

Comment:
` createWorkspaceFromConfig` can be used, cause in stack's workspace config section - we can have no only commands defined.